catroid/src/main/res/values/strings.xml

```diff
@@ -61,16 +61,16 @@
 
     <string name="backpack_add">Add to backpack</string>
 
-    <string name="share_website_text">www.pocketcode.org</string>
+    <string name="share_website_text">share.catrob.at</string>
```

Comment:
Actually, I don't think that such urls should be in strings.xml, since these strings can all be translated for all human languages that we support. Not a good idea to have different "translations" for this url.